packages/icons/package.json

```diff
@@ -38,7 +38,7 @@
     "picocolors": "1.0.0",
     "prompts": "2.4.2",
     "svgo": "2.8.0",
-    "vite": "2.9.8"
+    "vite": "2.9.9"
```

Comment:
this change is just so both the icons and the react package use the same versions